### PR TITLE
Improve docs styling for code to be run in shell

### PR DESCRIPTION
**Pull request by ashmaroli**
### Current design:
jekyllrb com_docs_quickstart

### With this patch..:
localhost_4000_docs_quickstart

### CODE REVIEW

File: docs/_sass/_style.scss [hunk 1]
```diff
@@ -1136,11 +1137,26 @@ code.output {
     line-height: 1.25;
     font-weight: 700;
     content: "TERMINAL";
-    background: #7a7a7a;
+    text-align: center;
+    background: linear-gradient(#b3b3b3, #ededed);
```
Comment (pup) [inline docs/_sass/_style.scss]: imo this could be a solid color, to be consistent with the current MacOS styling

Comment (ashmaroli) [reply docs/_sass/_style.scss]: Agreed, but I wasn't shooting for an exact macOS style.. just a close replica..
Moreover we do use gradients elsewhere on `jekyllrb.com`.. ..

Comment (mattr-) [reply docs/_sass/_style.scss]: Could we see it with a solid color instead? I'm leaning that way as well.

**Comment by mattr-** [comment]
Nice work on this. I like that the terminal background is a bit darker which gives more contrast to the white text. I like that the title bar is a lighter color too.

I don't agree with the color change to the comments because they serve as instructions for what to do and now they're in danger of getting lost due to the lack of contrast. Could you tweak it such that they're still gray but just a lighter shade? I don't think it's necessary to go back to the blue but we definitely don't want to lose them.

**Comment by ashmaroli** [comment]
Looking around.., we already have a style similar to an OS chrome..

jekyllrb com_

**Comment by pup** [comment]
@ashmaroli okay, that's fair then

**Comment by DirtyF** [comment]
> Looking around.., we already have a style similar to an OS chrome..

👍 Let's just use the existing style then, we don't need the extra macOS buttons here.

**Comment by ashmaroli** [comment]
To use the existing style would need to modify generated markup as well.. instead incorporated that style into this patch..
...kept the mac buttons since it immediately signals an app window chrome..
...also used a lighter-gray for the comment strings..

localhost_4000_docs_quickstart_mod

**Comment by DirtyF** [comment]
I tested  and we can match the existing style without modifying the markup:

terminal

**Comment by ashmaroli** [comment]
@DirtyF  Feel free to update the branch as you see fit.. :slightly_smiling_face:

**Comment by DirtyF** [comment]
@jekyllbot: merge +site

**Pull request merged**